### PR TITLE
Fix: update readme

### PR DESCRIPTION
**Pull request by clindeman**
Replace all anaconda references with miniforge. Change hand drawn red circles with straight red lines in screenshots.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ The Flood Hazard Import Tool FHIT is used to download depth grids for use in Haz
 
 ## Requirements
 
-The Flood Hazard Import Tool requires Python, Hazpy, Miniforge, and an internet connection. See https://github.com/nhrap-dev/Environment-Setup on how to setup your environment before running FHIT. See the To Use section on how to install FHIT.
+The Flood Hazard Import Tool requires Hazus, Conda, and an internet connection. If you haven't already, follow this [README](https://github.com/nhrap-dev/Environment-Setup/tree/update-README) to setup your machine with miniforge and conda before trying to run FHIT.
```
Comment (be-cunning) [inline README.md]: Use an active tone. Change the second sentence to:

Follow the [README](https://github.com/nhrap-dev/Environment-Setup/tree/update-README) to setup your machine with miniforge and conda before running FHIT.

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ The Flood Hazard Import Tool FHIT is used to download depth grids for use in Haz
...
 
 ## To Use
 
-1. Download zip folder of tool from GitHub, unzip
+1. Download zip folder of tool from GitHub, unzip.
```
Comment (be-cunning) [inline README.md]: Change to this for clarity:

_Download zip folder of tool from GitHub, then unzip._

**Review by be-cunning — CHANGES_REQUESTED** [review]
Just needs a few sentence corrections - to use active tone.

Also, these commits are a 'chore', instead of a 'fix'.

File: README.md (hunk 1 above)
Comment (clindeman) [reply README.md]: Applied recommended active tone.

File: README.md (hunk 2 above)
Comment (clindeman) [reply README.md]: Applied recommendation for clarity.

**Pull request merged**